includes/class-cache.php

```
@@ -0,0 +1,430 @@
+        
+        // Save for 24 hours
+        set_transient('zippicks_cache_stats', $saved_stats, DAY_IN_SECONDS);
+    }
+    
+    /**
+     * Get cache info
+     *
+     * @return array
+     */
+    public function get_info() {
+        global $_wp_using_ext_object_cache;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$_wp_using_ext_object_cache'. (Unused Code Rules)

(UnusedLocalVariable)